backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -344,12 +338,10 @@ def notify_content_update(
                 translator,
             )
 
-            part1 = MIMEText(body_text, 'plain', 'utf-8')
```

Comment:
> @lebouquetin Should we not always attach a text version to an email ? How it will be displayed on "only text" mail client interfaces ?

I talked about this with a [well known debian engineer](https://raphaelhertzog.fr/) using mutt as email client, he told me that mutt is clever enough in order to interpret HTML mails.

The only interesting point I see in the article is about spam score. Do you have more links/explanations (ie with concrete results) about the way email spam score is impacted by text-version of emails?

Reply:
@lebouquetin We can read some things about it [source](https://www.dailystory.com/blog/plain-text-email/):

> Decreased spam score – While some older anti-spam platforms have the ability to flag HTML only versions of your email as spam, the real culprit is link and image to content ratios. And email with lots of images and links, but very little content, is a red flag. But sender score and properly configured SPF and DMARC are much more important as it relates to spam scoring.

> As an example, SPAM Assassin, a popular Open Source anti-spam platform includes a rule MIME_HTML_ONLY. This rules triggers when an email message “only has text/html MIME parts” and is missing a text/plain alternative.

What i suggest is to open an issue to add later a auto-generated text version.

Reply:
issue opened here about text email template, see #1422